README.md

```diff
@@ -11,6 +11,14 @@ AWS_ACCESS_KEY_ID=
 AWS_SECRET_ACCESS_KEY=
 AWS_REGION=us-east-1
 PRE_SHARED_KEY=
+MONITER_ENABLE=True
```

Comment:
don't forget to update the env sample file.

Reply:
another thing, typo `MONITER` should be `MONITOR`

Reply:
> another thing, typo `MONITER` should be `MONITOR`

Thank you for correcting a mistake in my low -level